README.md

```diff
@@ -80,6 +80,13 @@ After you've fiddled with it, rename it to `.cargo/config.toml` to enable it.
 This template uses [GitHub workflows](https://docs.github.com/en/actions/using-workflows) to run tests and build releases.
 See [Workflows](./docs/workflows.md) for more information.
 
+## Known Issues
+
+There are some known issues in Bevy that require some arcane workarounds.
+None of these should be hindering your jam, but if you run into them, their fix is often not obvious.
```

Comment:
I want to leave it in since it's a description of how "bad" these are. If read that Bevy has "known issues", I might wonder "Okay, so I cannot use it in practice?" or "So I will run into them instantly?" or "Can I just fix them on my own?"

Made the wording more wordy to not only specify jam: [`f325307` (#176)](https://github.com/TheBevyFlock/bevy_quickstart/pull/176/commits/f325307d4d941bfd5085e44d6bd8e41929c61ebe)
I do not want to say that "this will not hinder your project" in general, as that's not true. A commercial project will very quickly care about these things.

Reply:
It's also not exactly true because the web audio issue can definitely hinder your progress in a game jam. I had to spend a lot of time tuning bevy jam simulator to be performant enough on web with no multithreading so that audio would maybe be ok on weaker machines, behind a feature flag so that native gets the full experience.

Reply:
I get not wanting to paint Bevy in a bad light, but there _are_ issues and I'd rather be honest about it.

Reply:
Should I change it to just "If you run into them"?

Reply:
[`1a0597b` (#176)](https://github.com/TheBevyFlock/bevy_quickstart/pull/176/commits/1a0597bae51a1e0274a31e90937e8418cdae4464)